wormhole-connect/src/views/v2/Bridge/ReviewTransaction/WalletBalanceWarning.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import AlertBanner from 'components/v2/AlertBanner';
+import { Stack, Typography, useTheme } from '@mui/material';
+import { toFixedDecimals } from 'utils/balance';
+
+export type WalletBalanceWarningProps = {
+  isCheckingBalance: boolean;
+  isBalanceEnough: boolean;
+  walletBalance?: number;
+  networkCost?: number;
+  symbol?: string;
+};
+
+export default function WalletBalanceWarning({
+  isCheckingBalance,
+  isBalanceEnough,
+  walletBalance,
```

Comment:
Nitpick: Is this the token's balance in the wallet? In that case do you it'd be better to name it `tokenBalance`?

Reply:
leaving this one as is as per slack comments